tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -480,6 +480,102 @@ public function test_payment_fields_outputs_fields() {
+		update_option( WC_Stripe_Feature_Flags::CHECKOUT_SESSIONS_FEATURE_FLAG_NAME, $feature_flag ? 'yes' : 'no' );
+		add_filter( 'woocommerce_is_checkout', '__return_true' );
+
+		ob_start();
+		$gateway->payment_fields();
+		$output = ob_get_clean();
+
+		remove_filter( 'woocommerce_is_checkout', '__return_true' );
+		update_option( WC_Stripe_Feature_Flags::CHECKOUT_SESSIONS_FEATURE_FLAG_NAME, $original_feature_flag );
+		WC_Stripe_Helper::update_main_stripe_settings( $original_stripe_settings );
+
+		$selector_div = '<div id="wc-stripe-currency-selector" class="wc-stripe-currency-selector"></div>';
```

PHPMD (2.15.0)

[error] The variable $selector_div is not named in camelCase. (undefined)

(CamelCaseVariableName)